### ISSUE TITLE
"New card position" setting changes after desktop sync, despite Android's setting being newer

### ISSUE THREAD

**Issue by tbm** [ankidroid/Anki-Android#8028]
###### Reproduction Steps

1. Desktop: Set "New card position" to "New cards *before* reviews"
2. Android: Set "New card position" to "New cards *after* reviews"
3. Android: Perform sync
4. Desktop: Perform sync
5. Android: Perform Perform sync
6. Android: Check "New card position": it's now "New cards *before* reviews"

###### Expected Result

I noticed that "New card position" always changed after doing a sync in Android after doing a sync on desktop (desktop haad new cards *before* reviews whereas I always changed Android to *after*).

It seems that options are synced, but only one-way from desktop to Android.

Would it be possible to sync options two-way, taking into account where they were changed last?

If not, i.e. if desktop is always the "authoritative" source, this issue might at least be worth adding to the documentation.

###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
If I recall correctly there are some related issues about the global config not syncing correctly and it is not intentional, there are (again if memory serves) just some errors with regard to how collection conf is synced. Not sure where exactly this option is stored but that may be relevant

**Comment by david-allison-1** [comment]
@tbm What version of Anki Desktop are you on? 

This could be related to the following (and therefore not an AnkiDroid issue): 
* https://forums.ankiweb.net/t/why-is-my-sync-button-blue/2078/32
* https://github.com/ankitects/anki/commit/73679b03e73e4df23ad59fdccb6cfb2a7fb50a8e

**Comment by tbm** [comment]
I'm on 2.1.33, which is older than this.  I'll upgrade and re-test.

**Comment by tbm** [comment]
@david-allison-1 Thanks for pointing this out. I *think* this has been fixed.  It's a bit hard to say for a number of reasons (1. I wonder if there should be been a sync after step 1 in my instructions, but iirc I left that out on purpose; 2. also it seems that merely opening the preferences menu on desktop without making a change results in the old setting being saved again, which can lead to problems if you change on android, but this is really a corner case).

But generally, yes, things are working as expected yet.  (I think there are still some edge cases where you don't get the desired behaviour, but I have a hard time coming up with clear step-by-step instructions, so I doubt anyone would run into it in the real world.)

Thanks!

And thanks for being so responsive, both @david-allison-1 and @mikehardy!
